src/pallet/stake-exchange/processors/buy-order-completed.ts

```diff
@@ -79,13 +71,11 @@ export async function buyOrderCompleted(
         existingMember.unbondingEras === null &&
         (!existingMember.tokenAccount || existingMember.tokenAccount.balance <= 0n)
     ) {
-        const poolMemberRewards = await ctx.store.findBy<PoolMemberRewards>(PoolMemberRewards, {
-            member: { id: existingMember.id },
-        })
         const poolMember = await ctx.store.findOneByOrFail<PoolMember>(PoolMember, { id: existingMember.id })
-        await ctx.store.remove(poolMemberRewards)
-        await ctx.store.remove(poolMember)
+        poolMember.bonded = existingMember.tokenAccount?.balance || 0n
```

Comment:
```suggestion
        poolMember.bonded = existingMember.tokenAccount?.balance || 0n
```
bonded is not the same thing that tokenAccount balance... not sure how to handle this

Reply:
we can set it to 0n

Reply:
yeah I think it make sense, let just set it to 0